### PR TITLE
[hi] Localize content/hi/docs/contribute/new-content/new-features.md

### PR DESCRIPTION
**Pull request by amolmote**
Signed-off-by: amolmote <amolmote201@gmail.com>
Fixes subtopic issue #36468 [hi] Localize en/docs/contribute/new-content/new-features.md into  hi/docs/contribute/new-content/new-features.md
which is the sub-issue of #36455

### CODE REVIEW

File: content/hi/docs/contribute/new-content/new-features.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+card:
+  name: contribute
+  weight: 45
+  title:  Documenting a feature for a release
+---
+<!-- overview -->
+
+प्रत्येक प्रमुख कुबेरनेट्स रिलीज़ नई सुविधाएँ पेश करती है जिन्हें प्रलेखन की आवश्यकता होती है। नई रिलीज़ मौजूदा सुविधाओं और प्रलेखन में भी अपडेट लाती हैं (जैसे किसी सुविधा को अल्फा से बीटा में अपग्रेड करना)।
+
+आम तौर पर, सुविधा के लिए जिम्मेदार एसआईजी सुविधा के ड्राफ्ट प्रलेखन को `kubernetes/website` रिपॉजिटरी की उपयुक्त विकास शाखा को पुल अनुरोध के रूप में प्रस्तुत करता हैं, और एसआईजी डॉक्स टीम में कोई व्यक्ति संपादकीय प्रतिक्रिया प्रदान करता हैं, या ड्राफ्ट को सीधे संपादित करता हैं। इस खंड में दोनों समूहों द्वारा रिलीज के दौरान उपयोग की जाने वाली ब्रांचिंग परम्पराएं और प्रक्रिया शामिल हैं।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:16]:
```suggestion
आम तौर पर, सुविधा के लिए जिम्मेदार SIG सुविधा के ड्राफ्ट प्रलेखन को `kubernetes/website` रिपॉजिटरी की उपयुक्त विकास शाखा को पुल अनुरोध के रूप में प्रस्तुत करता हैं, और SIG डॉक्स टीम में कोई व्यक्ति संपादकीय प्रतिक्रिया प्रदान करता हैं, या ड्राफ्ट को सीधे संपादित करता हैं। इस खंड में दोनों समूहों द्वारा रिलीज के दौरान उपयोग की जाने वाली ब्रांचिंग परम्पराएं और प्रक्रिया शामिल हैं।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+
+<!-- body -->
+
+## प्रलेखीकरण योगदानकर्ताओं के लिए
+
+सामान्य तौर पर, प्रलेखीकरण योगदानकर्ता किसी रिलीज़ के लिए शुरू से ही सामग्री नहीं लिखते हैं।
+इसके बजाय, वे मसौदा प्रलेखन को परिष्कृत करने और इसे जारी करने के लिए, तैयार करने के लिए, एक नई सुविधा बनाने वाले एसआईजी के साथ काम करते हैं।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:25]:
```suggestion
इसके बजाय, वे मसौदा प्रलेखन को परिष्कृत करने और इसे जारी करने के लिए, तैयार करने के लिए, एक नई सुविधा बनाने वाले SIG के साथ काम करते हैं।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+प्रलेखन या सहायता के लिए एक सुविधा चुनने के बाद, इसके बारे में `#sig-docs` स्लैक चैनल में, साप्ताहिक एसआईजी डॉक्स मीटिंग में, या सीधे सुविधा एसआईजी द्वारा दायर पीआर पर पूछें। यदि आपको आगे बढ़ने की अनुमति दी गई हैं, तो आप [किसी अन्य व्यक्ति के पीआर में प्रतिबद्ध](/docs/contribute/review/for-approvers/#commit-into-another-persons-pr) में वर्णित तकनीकों में से एक का उपयोग करके पीआर में संपादित कर सकते हैं।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:27]:
```suggestion
प्रलेखन या सहायता के लिए एक सुविधा चुनने के बाद, इसके बारे में `#sig-docs` स्लैक चैनल में, साप्ताहिक SIG डॉक्स मीटिंग में, या सीधे सुविधा SIG द्वारा दायर PR पर पूछें। यदि आपको आगे बढ़ने की अनुमति दी गई हैं, तो आप [किसी अन्य व्यक्ति के PR में प्रतिबद्ध](/docs/contribute/review/for-approvers/#commit-into-another-persons-pr) में वर्णित तकनीकों में से एक का उपयोग करके PR में संपादित कर सकते हैं।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+### आगामी सुविधाओं के बारे में जानें
+
+आगामी सुविधाओं के बारे में जानने के लिए, साप्ताहिक एसआईजी रिलीज़ बैठक में भाग लें (आगामी बैठक के लिए [समुदाय](/community/) पेज देखें) और [कुबेरनेट्स/एसआईजी-रिलीज़](https://github. com/kubernetes/sig-release/)रिपॉजिटरी देखें। प्रत्येक रिलीज़ की [/एसआईजी-रिलीज़/ट्री/मास्टर/रिलीजेस/](https://github.com/kubernetes/sig-release/tree/master/releases) निर्देशिका में एक उप-निर्देशिका होती है। उप-निर्देशिका में एक रिलीज़ अनुसूची, रिलीज़ टिप्पणियाँ का एक ड्राफ्ट और रिलीज़ समूह में प्रत्येक व्यक्ति को सूचीबद्ध करने वाला एक प्रलेखन होता है।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:31]:
```suggestion
आगामी सुविधाओं के बारे में जानने के लिए, साप्ताहिक SIG रिलीज़ बैठक में भाग लें (आगामी बैठक के लिए [समुदाय](/community/) पेज देखें) और [कुबेरनेट्स/SIG -रिलीज़](https://github. com/kubernetes/sig-release/)रिपॉजिटरी देखें। प्रत्येक रिलीज़ की [/SIG-रिलीज़/ट्री/मास्टर/रिलीजेस/](https://github.com/kubernetes/sig-release/tree/master/releases) निर्देशिका में एक उप-निर्देशिका होती है। उप-निर्देशिका में एक रिलीज़ अनुसूची, रिलीज़ टिप्पणियाँ का एक ड्राफ्ट और रिलीज़ समूह में प्रत्येक व्यक्ति को सूचीबद्ध करने वाला एक प्रलेखन होता है।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+इस प्रलेखन में **सुविधा ट्रैकिंग शीट** का लिंक भी शामिल है, जो रिलीज में जाने के लिए निर्धारित सभी नई सुविधाओं के बारे में पता लगाने का आधिकारिक तरीका है।
+
+रिलीज़ समूह प्रलेखन सूचीबद्ध करता है कि प्रत्येक रिलीज़ भूमिका के लिए कौन ज़िम्मेदार है। यदि यह स्पष्ट नहीं है कि किसी विशिष्ट विशेषता या आपके प्रश्न के बारे में किससे बात करनी है, तो या तो अपना प्रश्न पूछने के लिए रिलीज़ बैठक में भाग लें, या रिलीज़ प्रमुख से संपर्क करें ताकि वे आपको पुनर्निर्देशित कर सकें।
+
+रिलीज़ नोट्स ड्राफ़्ट रिलीज़ के बारे में विशिष्ट विशेषताओं, परिवर्तनों, बहिष्करणों और अधिक के बारे में जानने के लिए एक अच्छी जगह हैं। रिलीज चक्र में देर तक सामग्री को अंतिम रूप नहीं दिया गया हैं, इसलिए सावधानी बरतें।
+
+### सुविधा ट्रैकिंग शीट
+
+सुविधा ट्रैकिंग शीट [कुबेरनेट्स रिलीज के लिए दिए गए](https://github.com/kubernetes/sig-release/tree/master/releases) प्रत्येक सुविधा को सूचीबद्ध करती हैं जिसे रिलीज के लिए योजनाबद्ध किया गया हैं।
+प्रत्येक पंक्ति वस्तु में सुविधा का नाम शामिल होता हैं,सुविधा के मुख्य गिटहब मुद्दे का लिंक,इसका स्थिरता स्तर (अल्फा, बीटा, या स्थिर),एसआईजी और इसे लागू करने के लिए जिम्मेदार व्यक्ति,चाहे उसे प्रलेखीकरण की आवश्यकता हो,
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:44]:
```suggestion
प्रत्येक पंक्ति वस्तु में सुविधा का नाम शामिल होता हैं,सुविधा के मुख्य गिटहब मुद्दे का लिंक,इसका स्थिरता स्तर (अल्फा, बीटा, या स्थिर),SIG और इसे लागू करने के लिए जिम्मेदार व्यक्ति,चाहे उसे प्रलेखीकरण की आवश्यकता हो,
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+या सुविधा के लिए एक मसौदा रिलीज नोट और क्या यह विलय कर दिया गया हैं। निम्नलिखित बातों का ध्यान रखें:
+
+- बीटा और स्थिर सुविधाएँ आमतौर पर अल्फा सुविधाओं की तुलना में एक उच्च प्रलेखीकरण की प्राथमिकता होती हैं।
+- ऐसी सुविधा का परीक्षण करना (और इसलिए प्रलेखन करना) कठिन है जिसे विलय नहीं किया गया है, या कम से कम इसके पीआर में सुविधा-पूर्ण माना जाता है।
+- यह निर्धारित करना कि किसी सुविधा को प्रलेखीकरण की आवश्यकता है या नहीं, एक मैन्युअल प्रक्रिया है। भले ही किसी सुविधा को आवश्यक दस्तावेज़ के रूप में चिह्नित न किया गया हो, आपको सुविधा का प्रलेखीकरण करने की आवश्यकता हो सकती है।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:49]:
```suggestion
- ऐसी सुविधा का परीक्षण करना (और इसलिए प्रलेखन करना) कठिन है जिसे विलय नहीं किया गया है, या कम से कम इसके PR में सुविधा-पूर्ण माना जाता है।
- यह निर्धारित करना कि किसी सुविधा को प्रलेखीकरण की आवश्यकता है या नहीं, एक मैन्युअल प्रक्रिया है। भले ही किसी सुविधा को आवश्यक प्रलेखन के रूप में चिह्नित न किया गया हो, आपको सुविधा का प्रलेखीकरण करने की आवश्यकता हो सकती है।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+## डेवलपर्स या अन्य एसआईजी सदस्यों के लिए
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:51]:
```suggestion
## डेवलपर्स या अन्य SIG सदस्यों के लिए
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+
+यह खंड अन्य कुबेरनेट्स एसआईजी के सदस्यों के लिए सूचना है जो रिलीज के लिए नई सुविधाओं का प्रलेखीकरण करते हैं।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:53]:
```suggestion
यह खंड अन्य कुबेरनेट्स SIG के सदस्यों के लिए सूचना है जो रिलीज के लिए नई सुविधाओं का प्रलेखीकरण करते हैं।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+
+यदि आप कुबेरनेट्स के लिए एक नई सुविधा विकसित करने वाले एसआईजी के सदस्य हैं, तो आपको यह सुनिश्चित करने के लिए एसआईजी डॉक्स के साथ काम करने की आवश्यकता है कि आपकी सुविधा रिलीज़ के लिए समय पर प्रलेखित है। शेड्यूलिंग विवरण और समय सीमा को सत्यापित करने के लिए [सुविधा ट्रैकिंग स्प्रेडशीट](https://github.com/kubernetes/sig-release/tree/master/releases) की जाँच करें या कुबेरनेट्स के `#sig-release` स्लैक चैनल में जाँच करें।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:55]:
```suggestion
यदि आप कुबेरनेट्स के लिए एक नई सुविधा विकसित करने वाले SIG के सदस्य हैं, तो आपको यह सुनिश्चित करने के लिए SIG डॉक्स के साथ काम करने की आवश्यकता है कि आपकी सुविधा रिलीज़ के लिए समय पर प्रलेखित है। शेड्यूलिंग विवरण और समय सीमा को सत्यापित करने के लिए [सुविधा ट्रैकिंग स्प्रेडशीट](https://github.com/kubernetes/sig-release/tree/master/releases) की जाँच करें या कुबेरनेट्स के `#sig-release` स्लैक चैनल में जाँच करें।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+
+### स्थानधारक पीआर खोलें
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:57]:
```suggestion
### स्थानधारक PR खोलें
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 11]
```diff
@@ -0,0 +1,81 @@
...
+
+1. `kubernetes/website` रिपॉजिटरी में `dev-{{< skew nextMinorVersion >}}` शाखा के खिलाफ एक **draft** पुल अनुरोध खोलें, एक छोटी प्रतिबद्धता के साथ जिसे आप बाद में संशोधित करेंगे। ड्राफ़्ट पुल अनुरोध बनाने के लिए, पुल अनुरोध बनाएँ, ड्रॉप-डाउन का उपयोग करें और **Create Draft Pull Request** चुनें, फिर **Draft Pull Request** पर क्लिक करें।
+2. [कुबेरनेट्स/कुबेरनेट्स](https://github.com/kubernetes/kubernetes) पीआर(एस) और [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दों के लिंक शामिल करने के लिए पुल अनुरोध विवरण संपादित करें।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:60]:
```suggestion
2. [कुबेरनेट्स/कुबेरनेट्स](https://github.com/kubernetes/kubernetes) PR(s) और [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दों के लिंक शामिल करने के लिए पुल अनुरोध विवरण संपादित करें।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+3. पीआर के लिंक के साथ संबंधित [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दे पर एक टिप्पणी छोड़ें ताकि इस रिलीज को प्रबंधित करने वाले प्रलेखन व्यक्ति को सूचित किया जा सके कि सुविधा प्रलेखन आ रहे हैं और रिलीज के लिए ट्रैक किया जाना चाहिए।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:61]:
```suggestion
3. PR के लिंक के साथ संबंधित [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दे पर एक टिप्पणी छोड़ें ताकि इस रिलीज को प्रबंधित करने वाले प्रलेखन व्यक्ति को सूचित किया जा सके कि सुविधा प्रलेखन आ रहे हैं और रिलीज के लिए ट्रैक किया जाना चाहिए।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 13]
```diff
@@ -0,0 +1,81 @@
...
+
+यदि आपकी सुविधा को किसी प्रलेखीकरण परिवर्तन की आवश्यकता नहीं है, तो सुनिश्चित करें कि सिग-रिलीज़ टीम इसे `#sig-release` स्लैक चैनल में उल्लेख करना जानती है। यदि सुविधा को प्रलेखीकरण की आवश्यकता है लेकिन पीआर नहीं बनाया गया है, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:63]:
```suggestion
यदि आपकी सुविधा को किसी प्रलेखीकरण परिवर्तन की आवश्यकता नहीं है, तो सुनिश्चित करें कि SIG-रिलीज़ टीम इसे `#sig-release` स्लैक चैनल में उल्लेख करना जानती है। यदि सुविधा को प्रलेखीकरण की आवश्यकता है लेकिन PR नहीं बनाया गया है, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 14]
```diff
@@ -0,0 +1,81 @@
...
+
+### समीक्षा के लिए तैयार पीआर
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:65]:
```suggestion
### समीक्षा के लिए तैयार PR
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 15]
```diff
@@ -0,0 +1,81 @@
...
+
+तैयार होने पर, अपने स्थानधारक पीआर को सुविधा प्रलेखन से भरें और पीआर की स्थिति को ड्राफ्ट से **ready for review** में बदलें। पुल अनुरोध को समीक्षा के लिए तैयार के रूप में चिह्नित करने के लिए, मर्ज बॉक्स पर नेविगेट करें और **Ready for review** पर क्लिक करें।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:67]:
```suggestion
तैयार होने पर, अपने स्थानधारक PR को सुविधा प्रलेखन से भरें और पीआर की स्थिति को ड्राफ्ट से **ready for review** में बदलें। पुल अनुरोध को समीक्षा के लिए तैयार के रूप में चिह्नित करने के लिए, मर्ज बॉक्स पर नेविगेट करें और **Ready for review** पर क्लिक करें। 
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+यदि आपकी सुविधा को किसी प्रलेखीकरण परिवर्तन की आवश्यकता नहीं है, तो सुनिश्चित करें कि सिग-रिलीज़ टीम इसे `#sig-release` स्लैक चैनल में उल्लेख करना जानती है। यदि सुविधा को प्रलेखीकरण की आवश्यकता है लेकिन पीआर नहीं बनाया गया है, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
+
+### समीक्षा के लिए तैयार पीआर
+
+तैयार होने पर, अपने स्थानधारक पीआर को सुविधा प्रलेखन से भरें और पीआर की स्थिति को ड्राफ्ट से **ready for review** में बदलें। पुल अनुरोध को समीक्षा के लिए तैयार के रूप में चिह्नित करने के लिए, मर्ज बॉक्स पर नेविगेट करें और **Ready for review** पर क्लिक करें। 
+
+अपनी सुविधा का वर्णन करने और इसका उपयोग करने के तरीके के बारे में बताने की पूरी कोशिश करें। यदि आपको अपने प्रलेखीकरण को संरचित करने में सहायता की आवश्यकता है, तो `#sig-docs` स्लैक चैनल में पूछें।
+
+जब आप अपनी सामग्री को पूरा करते हैं, तो आपकी सुविधा को सौंपा गया प्रलेखीकरण व्यक्ति इसकी समीक्षा करता है।
+तकनीकी सटीकता सुनिश्चित करने के लिए, सामग्री को संबंधित एसआईजी (एस) से तकनीकी समीक्षा की भी आवश्यकता हो सकती है।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:72]:
```suggestion
तकनीकी सटीकता सुनिश्चित करने के लिए, सामग्री को संबंधित SIG(s) से तकनीकी समीक्षा की भी आवश्यकता हो सकती है।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+सामग्री को रिलीज़ के लिए तैयार स्थिति में लाने के लिए उनके सुझावों का उपयोग करें।
+
+यदि आपकी सुविधा एक अल्फा या बीटा सुविधा है और एक सुविधा द्वार के पीछे है, तो सुनिश्चित करें कि आप इसे अपने पुल अनुरोध के हिस्से के रूप में [अल्फा/बीटा सुविधा द्वार](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-alpha-or-beta-features) तालिका में जोड़ते हैं। नए सुविधा द्वार के साथ सुविधा द्वार का विवरण भी आवश्यक है। यदि आपकी सुविधा को मार्गदर्शक या बहिष्कृत किया गया है, तो उसे उस तालिका से [स्नातक या पदावनत सुविधाओं के लिए सुविधा द्वार](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features) अल्फ़ा और बीटा इतिहास के साथ तालिका में ले जाना सुनिश्चित करें।
+
+यदि आपकी सुविधा को प्रलेखीकरण की आवश्यकता है और पहला मसौदा सामग्री प्राप्त नहीं होती है, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
+
+### सभी पीआर की समीक्षा और विलय के लिए तैयार
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:79]:
```suggestion
### सभी PR की समीक्षा और विलय के लिए तैयार
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+
+यदि आपके पीआर को अभी तक रिलीज की समय सीमा तक `dev-{{< skew nextMinorVersion >}}` शाखा में विलय नहीं किया गया है, तो रिलीज को प्रबंधित करने वाले डॉक्स व्यक्ति के साथ काम करें ताकि इसे समय सीमा तक प्राप्त किया जा सके। यदि आपकी सुविधा को प्रलेखीकरण की आवश्यकता है और प्रलेखन तैयार नहीं हैं, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
```
Comment (amolmote) [inline content/hi/docs/contribute/new-content/new-features.md:81]:
```suggestion
यदि आपके PR को अभी तक रिलीज की समय सीमा तक `dev-{{< skew nextMinorVersion >}}` शाखा में विलय नहीं किया गया है, तो रिलीज को प्रबंधित करने वाले डॉक्स व्यक्ति के साथ काम करें ताकि इसे समय सीमा तक प्राप्त किया जा सके। यदि आपकी सुविधा को प्रलेखीकरण की आवश्यकता है और प्रलेखन तैयार नहीं हैं, तो सुविधा को अभियान के महत्त्वपूर्ण लक्ष्य से हटाया जा सकता है।
```

File: content/hi/docs/contribute/new-content/new-features.md [hunk 19]
```diff
@@ -60,7 +60,7 @@ card:
 2. [कुबेरनेट्स/कुबेरनेट्स](https://github.com/kubernetes/kubernetes) PR(s) और [कुबेरनेट्स/एन्हांसमेंट्स](https://github.com/kubernetes/enhancements) मुद्दों के लिंक शामिल करने के लिए पुल अनुरोध विवरण संपादित करें।
```
Comment (divya-mohan0209) [inline content/hi/docs/contribute/new-content/new-features.md:60]: I don't know if we should be localizing pull request here.

**Review by divya-mohan0209 — COMMENTED** [review]
Slight nit otherwise LGTM

File: content/hi/docs/contribute/new-content/new-features.md (hunk 19 above)
Comment (amolmote) [reply content/hi/docs/contribute/new-content/new-features.md:60]: I think also as this is name of our repository, let's see what others are thinking about this till now I will rebase all these commits.

**Comment by amolmote** [comment]
@divya-mohan0209, please review I have made all the possible changes.

**Comment by amolmote** [comment]
@bishal7679, have a look.

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
@amolmote Please update your fork's branch with the latest content since it doesn't seem to be updated and resubmit for review!

**Comment by amolmote** [comment]
Hi @divya-mohan0209 , due to change in head I am unable to update this branch. created new PR. could you pls review this: https://github.com/kubernetes/website/pull/39197. same issue happened with my other PR's also I am trying to update it one by one.

**Comment by amolmote** [comment]
closing this PR now

**Pull request closed without merging**